os/binfmt/binfmt_loadbinary.c

```diff
@@ -193,15 +190,15 @@ int load_binary(int binary_idx, FAR const char *filename, load_attr_t *load_attr
 		uint8_t nregion = mpu_get_nregion_info(MPU_REGION_COMMON_BIN);
 #ifdef CONFIG_OPTIMIZE_APP_RELOAD_TIME
 		/* Get MPU register values for MPU regions */
-		mpu_get_register_config_value(&com_bin_mpu_regs[0], nregion - 3, (uintptr_t)bin->alloc[ALLOC_TEXT], bin->textsize, true,  true);
-		mpu_get_register_config_value(&com_bin_mpu_regs[3], nregion - 2, (uintptr_t)bin->alloc[ALLOC_RO],   bin->rosize,   true,  false);
-		mpu_get_register_config_value(&com_bin_mpu_regs[6], nregion - 1, (uintptr_t)bin->alloc[ALLOC_DATA], bin->ramsize,  false, false);
+		mpu_get_register_config_value(&bin->cmn_mpu_regs[0], nregion - 3, (uintptr_t)bin->alloc[ALLOC_TEXT], bin->textsize, true,  true);
```

Comment:
In case of common bin, `bin` is `g_lib_binp`, not `load_attr->binp`. Because binary manager calls load_binary with null for 3rd argu. Right? 
If this is right, I suggest minor change.
When you call binary_manager_load_binary, let's give addr of `g_lib_binp` instead of null. Then we can keep the same usages of the function and can remove if condition here. It's difficult to know exact content.

Reply:
Is it right `g_lib_binp` for common bin? then how about changing the name?